MAUI/Popup/modal-window.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Modal Window in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Modal Window support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.

```

Comment:
description: Learn here all about Modal Window support in the Syncfusion .NET MAUI Popup (SfPopup) control, its elements, and more.